doc/modules/mod_roster.md

```diff
@@ -19,3 +19,20 @@ It can sometimes become quite a heavyweight feature, so there is an option to di
                {store_current_id, true}
              ]}
 ```
+
+### Metrics
+
+If you'd like to learn more about metrics in MongooseIM, please visit [MongooseIM metrics](../operation-and-maintenance/Mongoose-metrics.md) page.
+
+| Name | Type | Description (when it gets incremented) |
+| ---- | ---- | -------------------------------------- |
+| `[global, backends, mod_roster, read_roster_version]` | histogram | Time to fetch version of a user's roster. |
+| `[global, backends, mod_roster, write_roster_version]` | histogram | Time to store version of a user's roster. |
+| `[global, backends, mod_roster, get_roster]` | histogram | Time to fetch a user's roster. |
+| `[global, backends, mod_roster, get_roster_entry]` | histogram | Time to fetch a specific roster entry. |
+| `[global, backends, mod_roster, get_roster_entry_t]` | histogram | Time to fetch a specific roster entry inside a transaction. |
+| `[global, backends, mod_roster, get_subscription_lists]` | histogram | Time to fetch a subscription list of a user. |
+| `[global, backends, mod_roster, roster_subscribe_t]` | histogram | Time to update subscription status between users inside a transaction. |
```

Comment:
| `[global, backends, mod_roster, roster_subscribe_t]` | histogram | Time it takes to update the subscription status between users inside a transaction. |